<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPackage</class>
 <widget class="QDialog" name="AddPackage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Package</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>20</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 6pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
      <property name="text">
       <string>Package Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtPackageName">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 7pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="txtDescription">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
      <property name="text">
       <string>Amount</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtAmount">
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
font: 9pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #555;
font: 7pt &quot;Wide Latin&quot;;
background: transparent;
color: black;
</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
